src/github.com/getlantern/lantern-ui/app/partials/error.html

```diff
@@ -1,47 +1,47 @@
-<div id="error" ng-controller="ErrorCtrl" ng-show="showError && !showMobileAd">
-  <div id="oops">
-    {{ 'ERROR_OOPS' | translate }}
-  </div>
-  <div id="error-description">
-    {{ 'ERROR_DESCRIPTION' | translate }}
-  </div>
-  <div id="error-reasons">
-    <ol>
-      <li ng-show="!model.settings.systemProxy">
-        <div>{{ 'PROXY_OFF_ISSUE' | translate }}</div>
-        <div><a ng-click="toggleShowExtensionHelp()">{{ 'WHAT_DO' | translate }}</a></div>
-        <div class="error-action ng-hide" ng-show="showExtensionHelp">
-          {{ 'PROXY_OFF_HELP' | translate }}
-        </div>
-      </li>
-      <li>
-        <div>{{ 'BROWSER_EXTENSION_ISSUE' | translate }}</div>
-        <div><a ng-click="toggleShowExtensionHelp()">{{ 'WHAT_DO' | translate }}</a></div>
-        <div class="error-action ng-hide" ng-show="showExtensionHelp">
-          {{ 'EXTENSION_HELP' | translate }}
-        </div>
-      </li>
-      <li>
-        <div>{{ 'XUNLEI_ISSUE' | translate }}</div>
-        <div><a ng-click="toggleShowXunleiHelp()">{{ 'WHAT_DO' | translate }}</a></div>
-        <div class="error-action ng-hide" ng-show="showXunleiHelp">
-          {{ 'XUNLEI_HELP' | translate }}
-        </div>
-      </li>
-      <li>
-        <div>{{ 'ANOTHER_ISSUE' | translate }}</div>
-        <div><a ng-click="toggleShowConnectionHelp()">{{ 'WHAT_DO' | translate }}</a></div>
+<div ng-controller="ErrorCtrl">
+  <div id="error" ng-show="showError && !showMobileAd && !showBookmarks">
```

Comment:
This is a whitespace change except for the above two lines. These just separate the ng-controller div from the inner div (seems to be a good practice to separate the ng-show logic, for example, from the logic of the controller) but more importantly just adds the !showBookmarks line here to make sure we don't show the bookmarks if we're showing the error.